### ISSUE TITLE
Prom Metrics class within Teraslice needs to discover which url to associate with exported metrics

### ISSUE THREAD

**Issue by busma13** [terascope/teraslice#3748]
While working on #3743 we have discovered that the `url` label was missed in the move from the external `teraslice-exporter` to the one built into `terafoundation`.

The url where an instance of teraslice lives on the network is controlled at the k8s level (if I understand this correctly), so either needs to be provided as an environmental variable or added as a field to the terafoundation. If added to the terafoundation, do we include `prom_metrics` in the key? This makes sense with our current use case, but the url could be needed in other contexts in the future that are not prom metrics related.

Another option is to have the dashboard add the url labels to each metric after they are scraped by grabbing the url from the Ingress.

**Comment by godber** [comment]
This should be fixed in the v2.3.2 release.

**Comment by busma13** [comment]
I added `prom_metrics_display_url` to the terafoundation for ts-dev1 and the label shows properly in all metrics.
```
teraslice_slices_processed{name="teraslice-dev1",url="https://ts-dev1.tera4.lan"} 1457
```
